packages/graphql-eslint-rules/src/__tests__/__snapshots__/operation-naming-conventions.test.ts.snap

```diff
@@ -6,6 +6,11 @@ exports[` 1`] = `
 `;
 
 exports[` 2`] = `
-"> 1 | query wrongNameQuery { user { id name } }
-    | ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^ Filename should end with the operation name (query/mutation/subscription) e.g. foo-query.graphql"
+"> 1 | query GraphqlEslintRulesUserQuery { user { id name } }
+    | ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^ Filename should start with the component root directory name \\"graphql-eslint-rules\\""
```

Comment:
Can you CamelCase the package name in the output so they know how exactly it is expected?